puppet/modules/scaleway/manifests/cron.pp

```diff
@@ -0,0 +1,24 @@
+# Deploys a cronjob which calls the Scaleway API to
+# snapshot a server. Does not need to be on the Scaleway host.
+#
+define scaleway::cron (
+  String $api_key = 'NA',
+  String $org_id  = 'NA',
+  Boolean $ensure = present
```

Comment:
nit: we like trailing commas. Also think this should be `Enum['present', 'absent']` rather than a `Boolean`.

Reply:
Show how long it's been since I wrote any serious Puppet. On it.